content/en/docs/reference/command-line-tools-reference/feature-gates/strict-cos-enforcement-for-VAP.md

```diff
@@ -0,0 +1,15 @@
+---
+title: StrictCostEnforcementForVAP
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
```

Comment:
Is this an _alpha_ or _beta_ feature? Looking at the code, it appears to be beta. Apologies if I misunderstood or if there's any confusion on my end.

	genericfeatures.StrictCostEnforcementForVAP: {Default: false, PreRelease: featuregate.Beta},
	
_[Referring to kubernetes/kubernetes/pkg/features/kube_features.go#L1300 (here)](https://github.com/kubernetes/kubernetes/blob/6fc0a69044f1ac4c13841ec4391224a2df241460/pkg/features/kube_features.go#L1300)_

Reply:
Ohh I forgot we agreed to introduce this as beta for this special fg. Thanks